example-utils/integration-test-utils/src/up_client_foo.rs

```diff
@@ -0,0 +1,413 @@
+/********************************************************************************
```

Comment:
I'll point this out myself -- this is a bit of poor form. It _works_, but there's a fair bit of refactoring that could be done here to improve the code quality imho.

Reply:
Fixed this!